php/recipes/module-poppler-pdf.rb

```diff
@@ -1,3 +1,12 @@
-include_recipe 'php::dependencies-ppa'
+apt_repository 'poppler' do
+  uri          node['poppler']['package_uri']
+  arch         node['poppler']['package_arch']
```

Comment:
Look for `is_amd?` usage in the cookbooks and try not to hard-code the arch. Or is it only available for `amd64`?